regen_doc.g

```diff
@@ -36,6 +36,41 @@ else
     fi;
 fi;
 
+ListOfUnusedMethods := function()
+    local unusedMethods, methodRecords, methodRecordsNames, collection,
+        collectionName, i, methodName;
+    unusedMethods := [];
+    methodRecords := [FindHomMethodsGeneric, FindHomMethodsPerm,
+                      FindHomMethodsMatrix, FindHomMethodsProjective];
+    methodRecordsNames := ["FindHomMethodsGeneric", "FindHomMethodsPerm",
+                           "FindHomMethodsMatrix", "FindHomMethodsProjective"];
+    for i in [1..4] do
+        collection := methodRecords[i];
+        collectionName := methodRecordsNames[i];
+        for methodName in RecNames(collection) do
+            if IsEmpty(DbsWhichUseMethod(collection, methodName)) then
```

Comment:
Don't you get a warning about accessing an undefined global when you read this into a fresh GAP session, seeing as how `DbsWhichUseMethod` is only defined *after* this function?

Reply:
fixed